### ISSUE TITLE
Add to F-Droid

### ISSUE THREAD

**Issue by pizzamaker** [PicFrame/picframe#4]
It would be lovely to have this on F-Droid. Here's how to do it: https://f-droid.org/contribute/

**Comment by jswetzen** [comment]
Yes please! If this is still working I'd love to try it out, but I don't have Google Play store...
